src/tasks/TaskContext.js

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useContext, useState } from 'react';
+import * as taskService from '../services/taskService';
+
+const TaskContext = createContext();
+
+export function TaskProvider({ children }) {
```

Comment:
Do you use this Provider?  
because you also store the tasks in the `tasks/Task` component.